### PR TITLE
#122 Get `front` bot to 100% test coverage

### PR DESCRIPTION
**Pull request by novokrest**
Related to #122.

### CODE REVIEW

File: zulip_bots/zulip_bots/bots/front/test_front.py [hunk 1]
```diff
@@ -28,27 +34,53 @@ def test_archive(self) -> None:
             with self.mock_http_conversation('archive'):
                 self.verify_reply('archive', "Conversation was archived.")
 
+    def test_archive_error(self) -> None:
```
Comment (neiljp) [inline zulip_bots/zulip_bots/bots/front/test_front.py]: This increase in coverage is good to see, but a lot of these tests seem very similar; perhaps consider a helper function? Each test is very short, so is legible as it is, but the patterns suggest refactoring, particularly with all the error replies being identical.

Comment (showell) [reply zulip_bots/zulip_bots/bots/front/test_front.py]: I agree that a helper function is worthwhile here.  It's fine to just do this an additional commit that sweeps all the functions that follow the same pattern and does something more like `assert_bot_error(fixture='archive_error', message='Something went wrong')`

Comment (novokrest) [reply zulip_bots/zulip_bots/bots/front/test_front.py]: @neiljp I introduced helper function `_test_command_error` to test command's error.

**Comment by showell** [comment]
Merged, thanks @novokrest!

**Pull request merged**